game.py

```diff
@@ -0,0 +1,90 @@
+from etc.utils import Color, read_words, get_random_word, is_valid_word
+from collections import defaultdict
+from etc.consts import MAX_ATTEMPTS, INDEX_NOT_FOUND
+
+
+class WordleGame:
+    words = read_words()
+
+    def __init__(self):
+        self.__goal_word = ""
```

Comment:
Don't use dunder, use `_goal_word`.

Reply:
Done, but 2 underscores = private visibility and 1 underscore = protected no? 
what am I missing?

Reply:
See the video in the other comment that we discussed this.